### PR TITLE
Update PGS.ts

### LINKED ISSUE #2360: Wrong page of book linked on Rectangle Geometry
Hello! 

When I view the Rectangle Geometry page, and click on the link to the book that the module is based off, it takes me to a page (page 33) after the section about Rectangle Geometry (page 31). I couldn't find where these Resources links are stored, so I couldn't send a PR.

### PR DESCRIPTION
**Pull request by nwatx**
_If any of the below doesn't apply to this Pull Request, mark the checkbox as
done._

- [x] I have tested my code
- [x] I have followed the code style guidelines mentioned
      [here](https://usaco.guide/general/adding-solution/#code-conventions)
- [x] I have properly formatted the files according to the steps mentioned
      [here](https://usaco.guide/general/adding-solution#steps)

resolves #2360

### CODE REVIEW

**Review by SansPapyrus683 — APPROVED** [review]
can only assume this is correct

File: src/components/markdown/PGS.ts [hunk 1]
```diff
@@ -26,9 +26,9 @@ export default {
     '6.2': 31,
     '6.3': 31,
     '7': 33,
-    '7.1': 33,
-    '7.2': 33,
-    '7.3': 34,
+    '7.1': 31,
```
Comment (bqi343) [inline src/components/markdown/PGS.ts:29]: no way this is correct? the numbers are no longer in increasing order ...

**Review by bqi343 — CHANGES_REQUESTED** [review]

File: src/components/markdown/PGS.ts (hunk 1 above)
Comment (nwatx) [reply src/components/markdown/PGS.ts:29]: it's no longer in increasing order, but it's [correct](https://darrenyao.com/usacobook/cpp.pdf#page=31), I believe the others have to be updated as well

Comment (nwatx) [reply src/components/markdown/PGS.ts:29]: i'll look into it more i think a shift in pages occurred

Comment (nwatx) [reply src/components/markdown/PGS.ts:29]: er ok it seems like all the pages are shifted by a non-constant amount, do you want me to renumber the entire file

Comment (bqi343) [reply src/components/markdown/PGS.ts:29]: Yes. I believe that I copied the table of contents and used a Python script to extract the section and page numbers, but I'm not sure where it is anymore.

**Comment by bqi343** [comment]
Could you format this the same way as it was originally? also, looks like some things are missing for cpp.

**Review by bqi343 — APPROVED** [review]

**Pull request merged**